packages/glow-react/package.json

```diff
@@ -9,19 +9,25 @@
     "src/**/*"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "npm run build:js && npm run build:css",
+    "build:js": "tsc",
+    "build:css": "sass src/styles/index.scss:dist/styles.css",
     "release": "tsc && yarn version && npm publish --access public"
```

Comment:
I think we need to build both js and css before releasing, right?

Reply:
Right! Good catch

Reply:
Also it's interesting that we combine npm and yarn in that script and use pnpm for the rest haha

Have you looked into yarn v2? I used in the past and liked it a lot

Reply:
Haha I think we should just use `pnpm` but I don't know what all the commands are for like version / publish.